src/dmd/mtype.h

```diff
@@ -767,34 +768,40 @@ enum AliasThisRec
     RECtracingDT = 0x8  // mark in progress of deduceType
 };
 
-class TypeStruct : public Type
+class TypeAggregate : public Type
 {
 public:
-    StructDeclaration *sym;
+    AggregateDeclaration* asym;
     AliasThisRec att;
+    Dsymbol* toDsymbol(Scope* sc);
+    MATCH constConv(Type* to);
+    uint8_t deduceWild(Type* t, bool isRef);
+    void accept(Visitor *v) { v->visit(this); }
+};
+
+class TypeStruct : public TypeAggregate
+{
+public:
     bool inuse;
 
-    static TypeStruct *create(StructDeclaration *sym);
-    const char *kind();
-    d_uns64 size(const Loc &loc);
-    unsigned alignsize();
-    TypeStruct *syntaxCopy();
-    Dsymbol *toDsymbol(Scope *sc);
+    static TypeStruct* create(StructDeclaration* sym);
+    const char* kind();
+    d_uns64 size(const Loc& loc);
+    uint32_t alignsize();
+    TypeStruct* syntaxCopy();
```

Comment:
No need to make whitespace changes (also unsigned).

Reply:
Yeah, I copied from frontend.h. Do we have a `clang-format` config?

Reply:
> Do we have a clang-format config?

No, we don't.